front/src/applications/operationalStudies/helpers/TrainProjectionLazyLoader.ts

```diff
@@ -89,17 +98,32 @@ export default class TrainProjectionLazyLoader {
           )
         )
         .unwrap();
+
+      trainScheduleoccupancyBlocksPromise = this.options
+        .dispatch(
+          osrdEditoastApi.endpoints.postTrainScheduleOccupancyBlocks.initiate({
+            occupancyBlockForm: {
+              infra_id: this.options.infraId,
+              path: this.options.path,
+              ids: rawTrainScheduleIds,
+              electrical_profile_set_id: this.options.electricalProfileSetId,
```

Comment:
nit : does the this.option.[...] shall be split using `const { infraId, path, electricalProfileSetId} = this.options`